src/tribler.sh

```diff
@@ -2,8 +2,8 @@
 # Run Tribler from source tree
 
 script_path() {
-  DIR="${1%/*}"
-  (cd "$DIR" && echo "$(pwd -P)")
+  FILE="$(readlink -f "$1")"
```

Comment:
While using `readlink -f` makes it possible to run tribler.sh from any path, the command is not available on macOS which is why we avoided using it earlier.

Reply:
readlink must have been added for me by brew, I'll make it optional in the PR.